google/cloud/storage/bucket.py

```diff
@@ -1244,7 +1244,7 @@ def copy_blob(
         destination_bucket,
         new_name=None,
         client=None,
-        preserve_acl=True,
+        preserve_acl=None,
```

Comment:
Leave default value alone, instead update documentation that this parameter is not functional and label it deprecated. Do not emit a warning only add the deprecation documentation. Additionally, provide a snippet of how to copy ACLs over from one object to another with the existing client.